### PR TITLE
Added Microsoft OS files as well as content updates based on feedback

### PR DESCRIPTION
**Pull request by DirkSonguer**
Added required and recommended files to the repository:

- Microsoft OS Code of Conduct
- Microsoft recommended documentation guide

A lot of content rewrites based on the documentation guide.

Also added content feedback from the community.

### CODE REVIEW

File: documentation/deployment-devops.md [hunk 1]
```diff
@@ -0,0 +1,130 @@
...
+
+Click on Pipelines from the left-nav and then select "Create Pipeline".
+
+* From the "Where is your code?" screen, choose Github
+  * You may see a screen asking for authentication: "Authenticate to authorize access"
+* From the "Select a repository" screen, select the fork created above
+  * Select "Approve & Install Azure Pipelines" if required
+* From the "review your pipeline" screen, click the down-arrow next to Run and click "Save" - note that a number of variables need to be added before the first run
+
+### Update pipeline target condition
```
Comment (stevebus) [inline documentation/deployment-devops.md]: We moved the target condition to an environment variable (line 106)..  so the entire "Update pipeline target condition" section goes away.

File: documentation/deployment-manual.md [hunk 2]
```diff
@@ -0,0 +1,180 @@
...
+
+Before any Edge modules can be deployed, it is necessary to build the module images using the Dockerfiles found in the repository. Once built, the images need to be placed into a container registry.
+
+Start by cloning [the github repository](https://github.com/AzureIoTGBB/iot-edge-offline-dashboarding) to your local machine.
+
+```bash
+git clone https://github.com/AzureIoTGBB/iot-edge-offline-dashboarding.git
+```
+
+Replace `{registry}` in the commands below with the container registry location on your machine (e.g. myregistry.azurecr.io).
```
Comment (stevebus) [inline documentation/deployment-manual.md]: the registry location is not "on your machine"...  it's generally going to be in azure..   maybe say "the container registry location created earlier" or something like that

File: documentation/deployment-devops.md [hunk 3]
```diff
@@ -0,0 +1,130 @@
...
+
+## Forking the repository
+
+The DevOps pipeline details for the sample are included in [the github repository](https://github.com/AzureIoTGBB/iot-edge-offline-dashboarding).
+
+ Do a GitHub [fork](https://help.github.com/en/github/getting-started-with-github/fork-a-repo) of the repository to your own workspace. After that, continue making changes to the pipeline configuration, for example changing the target conditions.
+
+## Setting up an Azure DevOps organization and project
+
+A Azure DevOps pipeline is always part of a [project](https://docs.microsoft.com/en-us/azure/devops/organizations/projects/create-project?view=azure-devops&tabs=preview-page), which is part of an [organization](https://docs.microsoft.com/en-us/azure/devops/organizations/accounts/create-organization?view=azure-devops). Follow the instructions on the given websites, but skip the 'Add a Repository to your Project' part since this is managed on GitHub.
```
Comment (stevebus) [inline documentation/deployment-devops.md]: typo:   "An" Azure DevOps pipeline

**Review by stevebus — COMMENTED** [review]
Added some review comments.  also, it doesn't show up in the 'diffs', but there is another typo in your "customize-sample-oee" file.  Under "Adding the Site Level Performance to the dashboard", the first sentence says 'followinf' instead of 'following'.    Overall good changes

File: documentation/deployment-devops.md (hunk 1 above)
Comment (DirkSonguer) [reply documentation/deployment-devops.md]: Done. See commit [0212d04088f5955795e04005bb2a2cedb1bd989b](https://github.com/DirkSonguer/iot-edge-offline-dashboarding/commit/0212d04088f5955795e04005bb2a2cedb1bd989b).

File: documentation/deployment-manual.md (hunk 2 above)
Comment (DirkSonguer) [reply documentation/deployment-manual.md]: Done. See commit [0212d04088f5955795e04005bb2a2cedb1bd989b](https://github.com/DirkSonguer/iot-edge-offline-dashboarding/commit/0212d04088f5955795e04005bb2a2cedb1bd989b).

File: documentation/deployment-devops.md (hunk 3 above)
Comment (DirkSonguer) [reply documentation/deployment-devops.md]: Done. See commit [0212d04088f5955795e04005bb2a2cedb1bd989b](https://github.com/DirkSonguer/iot-edge-offline-dashboarding/commit/0212d04088f5955795e04005bb2a2cedb1bd989b).

File: documentation/deployment-devops.md [hunk 4]
```diff
@@ -103,7 +83,7 @@ To make the pipeline as generic as possible, much of the config is supplied in t
 * AZURE_SUBSCRIPTION_ID: The ID of the used Azure subscription
 * GRAFANA_ADMIN_PASSWORD: The desired administrator password for the Grafana dashboard web app when deployed
 * IOT_HUB_NAME: The name of the connected Azure IoT Hub (short name, without the .azure-devices.net)
-* DEPLOYMENT_TARGET_CONDITION: The target condition to use for the deployment. For example, for the tag example above, use 'tags.dashboard=true'.
+* DEPLOYMENT_TARGET_CONDITION: The target condition to use for the deployment. This is in line with the target tags for the Edge box's device twin. Based on the tag used above, the valuewould be 'tags.dashboard=true'.
```
Comment (stevebus) [inline documentation/deployment-devops.md]: one small typo left...    'valuewould' above

Comment (DirkSonguer) [reply documentation/deployment-devops.md]: Done, see commit ae64f05

**Pull request merged**